packages/main/src/plugin/index.ts

```diff
@@ -450,6 +450,18 @@ export class PluginSystem {
     return configurationRegistry;
   }
 
+  protected loadEnterpriseTelemetry(container: Container): void {
+    const defaultConfiguration = container.get<DefaultConfiguration>(DefaultConfiguration);
+    const defaultTelemetryInfo = defaultConfiguration.getTelemetryInfo();
+    const lockedConfiguration = container.get<LockedConfiguration>(LockedConfiguration);
+    const lockedTelemetryInfo = lockedConfiguration.getTelemetryInfo();
+    const telemetry = container.get<Telemetry>(Telemetry);
+
+    if (defaultTelemetryInfo) telemetry.track(defaultTelemetryInfo.event, defaultTelemetryInfo.eventProperties);
```

Comment:
avoid inline if statements? Im unsure if we have linting (it's passing?) but would be easier to read.